service/controller/resource/release/default_k8sclient.go

```diff
@@ -0,0 +1,79 @@
+package release
```

Comment:
should this be somewhere else?

Reply:
I would put it in a folder like `service/controller/resource/release/test` or `service/controller/resource/release/testing`

Reply:
Using similar location to [aws operator](https://github.com/giantswarm/aws-operator/blob/ha-ma/service/internal/unittest/default_k8sclient.go).